.github/workflows/main.yml

```
@@ -53,13 +53,18 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version-file: ".node-version"
-          cache: yarn
+
```

Do we need to use `pnpm` here?  ref: https://github.com/pnpm/action-setup?tab=readme-ov-file#use-cache-to-reduce-installation-time
```suggestion
          cache: pnpm
```

---

@stephannv , good point!